lib/metanorma/iho/processor.rb

```diff
@@ -19,12 +19,13 @@ def fonts_manifest
         {
           "Arial" => nil,
           "Cambria Math" => nil,
-          "Courier" => nil,
+          "Courier New" => nil,
           "Fira Code" => nil,
-          "Source Sans Pro" => [ "Light" ],
+          "Source Sans Pro Light" => nil,
           "Source Serif Pro" => nil,
```

Comment:
@Intelligent2013 Do "Source Serif Pro" require some subfamilies? May be "Source Serif Pro Light"?

Reply:
no.

Reply:
Thanks :)